yarn-project/end-to-end/src/bench/client_flows/client_flows_benchmark.ts

```diff
@@ -35,16 +40,70 @@ import {
   deployAccounts,
 } from '../../fixtures/snapshot_manager.js';
 import { mintTokensToPrivate } from '../../fixtures/token_utils.js';
-import { type SetupOptions, setupCanonicalFeeJuice } from '../../fixtures/utils.js';
+import { type SetupOptions, setupCanonicalFeeJuice, setupSponsoredFPC } from '../../fixtures/utils.js';
 import { CrossChainTestHarness } from '../../shared/cross_chain_test_harness.js';
 import {
   FeeJuicePortalTestingHarnessFactory,
   type GasBridgingTestHarness,
 } from '../../shared/gas_portal_test_harness.js';
 
-const { E2E_DATA_PATH: dataPath } = process.env;
+const { E2E_DATA_PATH: dataPath, BENCHMARK_CONFIG } = process.env;
 
 export type AccountType = 'ecdsar1' | 'schnorr';
+export type FeePaymentMethodGetter = (wallet: Wallet) => Promise<FeePaymentMethod>;
+export type BenchmarkingFeePaymentMethod = 'bridged_fee_juice' | 'private_fpc' | 'sponsored_fpc';
+
+export type ClientFlowConfig = {
+  accounts: AccountType[];
+  feePaymentMethods: BenchmarkingFeePaymentMethod[];
+  recursions?: number[];
+};
+
+type ClientFlows = 'deployments' | 'transfers' | 'bridging' | 'amm';
+
+type ClientFlowsConfig = {
+  [key in ClientFlows]: ClientFlowConfig;
+};
+
+const KEY_FLOWS_CONFIG: ClientFlowsConfig = {
```

Comment:
This is very nice. I'd consider moving this to a separate config file, so that it's obvious at a glance that there's config and that there is where it lives.

Reply:
Done!